### ISSUE TITLE
How a flawed idea is teaching millions of kids to be poor readers | At a Loss for Words

### ISSUE THREAD

**Issue by guevara** [guevara/read-it-later#6958]
How a flawed idea is teaching millions of kids to be poor readers | At a Loss for Words<br>
<br>
https://ift.tt/39yhPjc<br>
<br>
Emily Hanford<br>
<br>
<div>
<div>
<aside>
<div><a href="https://www.apmreports.org/educate-podcast"><img width="100%" src="https://img.apmcdn.org/a66c9bce4bd0503c23f90dc9020bd6d23d919f17/widescreen/7de8ca-20171201-educate.jpg"></a></div>
<div>
<p>Listen to this audio documentary on the Educate podcast.</p>
<a href="#subscribe"><strong>Subscribe now</strong>.</a>
</div>
</aside>
<p>Molly Woodworth was a kid who seemed to do well at everything: good grades, in the gifted and talented program. But she couldn't read very well.</p>
<p>"There was no rhyme or reason to reading for me," she said. "When a teacher would dictate a word and say, 'Tell me how you think you can spell it,' I sat there with my mouth open while other kids gave spellings, and I thought, 'How do they even know where to begin?' I was totally lost."</p>
<p>Woodworth went to public school in Owosso, Michigan, in the 1990s. She says sounds and letters just didn't make sense to her, and she doesn't remember anyone teaching her how to read. So she came up with her own strategies to get through text.</p>
<p>Strategy 1: Memorize as many words as possible. "Words were like pictures to me," she said. "I had a really good memory."</p>
<p>Strategy 2: Guess the words based on context. If she came across a word she didn't have in her visual memory bank, she'd look at the first letter and come up with a word that seemed to make sense. Reading was kind of like a game of 20 Questions: What word could this be?</p>
<p>Strategy 3: If all else failed, she'd skip the words she didn't know.</p>
<p>Most of the time, she could get the gist of what she was reading. But getting through text took forever. "I hated reading because it was taxing," she said. "I'd get through a chapter and my brain hurt by the end of it. I wasn't excited to learn."</p>
<p>No one knew how much she struggled, not even her parents. Her reading strategies were her "dirty little secrets."</p>
<figure><img alt="Molly and Nora" src="https://img.apmcdn.org/2829ff5aaa364c683d44fa92dfe6d7e1132d9fda/uncropped/e94708-20190819-molly-and-nora.jpg">
<figcaption><span>Molly Woodworth (left) with her aunt, Nora Chahbazi, outside the Ounce of Prevention Reading Center in Flushing, Michigan.</span> <span>Emily Hanford | APM Reports</span></figcaption>
</figure>
<p>Woodworth, who now works in accounting,<a href="#footnote-content-1">1</a> says she's still not a very good reader and tears up when she talks about it. Reading "influences every aspect of your life," she said. She's determined to make sure her own kids get off to a better start than she did.</p>
<p>That's why she was so alarmed to see how her oldest child, Claire, was being taught to read in school.</p>
<aside>
<blockquote>
<p>As long as this disproven theory remains part of American education, many kids will likely struggle to learn how to read.</p>
</blockquote>
</aside>
<p>A couple of years ago, Woodworth was volunteering in Claire's kindergarten classroom. The class was reading a book together and the teacher was telling the children to practice the strategies that good readers use.</p>
<p>The teacher said, "If you don't know the word, just look at this picture up here," Woodworth recalled. "There was a fox and a bear in the picture. And the word was bear, and she said, 'Look at the first letter. It's a "b." Is it fox or bear?'"</p>
<p>Woodworth was stunned. "I thought, 'Oh my God, those are my strategies.' Those are the things I taught myself to look like a good reader, not the things that good readers do," she said. "These kids were being taught my dirty little secrets."</p>
<p>She went to the teacher and expressed her concerns. The teacher told her she was teaching reading the way the curriculum told her to.</p>
<p>Woodworth had stumbled on to American education's own little secret about reading: Elementary schools across the country are teaching children to be poor readers &mdash; and educators may not even know it.</p>
<p>For decades, reading instruction in American schools has been rooted in a flawed theory about how reading works, a theory that was debunked decades ago by cognitive scientists, yet remains deeply embedded in teaching practices and curriculum materials. As a result, the strategies that struggling readers use to get by &mdash; memorizing words, using context to guess words, skipping words they don't know &mdash; are the strategies that many beginning readers are taught in school. This makes it harder for many kids to learn how to read, and children who don't get off to a good start in reading find it difficult to ever master the process.<a href="#footnote-content-2">2</a></p>
<p>A shocking number of kids in the United States can't read very well. A third of all fourth-graders can't read at a basic level, and most students are still not proficient readers by the time they finish high school.</p>
<hr>
<p><strong>Percentage of U.S. fourth-graders below basic level in reading</strong></p>
<hr>
<p>When kids struggle to learn how to read, it can lead to a downward spiral in which behavior, vocabulary, knowledge and other cognitive skills are eventually affected by slow reading development.<a href="#footnote-content-3">3</a>&nbsp;A disproportionate number of poor readers become high school dropouts and end up in the criminal justice system.<a href="#footnote-content-4">4</a></p>
<p>The fact that a disproven theory about how reading works is still driving the way many children are taught to read is part of the problem. School districts spend hundreds of millions of taxpayer dollars on curriculum materials that include this theory. Teachers are taught the theory in their teacher preparation programs and on the job. As long as this disproven theory remains part of American education, many kids will likely struggle to learn how to read.</p>
<hr>
<p><strong>Percentage of U.S. 12th-graders proficient in reading</strong></p>
<hr>
<h3>The origins</h3>
<p>The theory is known as "three cueing." The name comes from the notion that readers use three different kinds of information &mdash; or "cues" &mdash; to identify words as they are reading.</p>
<p>The theory was first proposed in 1967, when an education professor named Ken Goodman presented a paper at the annual meeting of the American Educational Research Association in New York City.</p>
<p>In the paper,<a href="#footnote-content-5">5</a>&nbsp;Goodman rejected the idea that reading is a precise process that involves exact or detailed perception of letters or words. Instead, he argued that as people read, they make predictions about the words on the page using these three cues:</p>
<ul>
<li>
<p>graphic cues (what do the letters tell you about what the word might be?)</p>
</li>
<li>
<p>syntactic cues (what kind of word could it be, for example, a noun or a verb?)</p>
</li>
<li>
<p>semantic cues (what word would make sense here, based on the context?)</p>
</li>
</ul>
<p>Goodman concluded that:</p>
<p>Skill in reading involves not greater precision, but more accurate first guesses based on better sampling techniques, greater control over language structure, broadened experiences and increased conceptual development. As the child develops reading skill and speed, he uses increasingly fewer graphic cues.</p>
<p>Goodman's proposal became the theoretical basis for a new approach to teaching reading that would soon take hold in American schools.</p>
<p>Previously, the question of how to teach reading had focused on one of two basic ideas.</p>
<p>One idea is that reading is a visual memory process. The teaching method associated with this idea is known as "whole word." The whole word approach was perhaps best embodied in the "Dick and Jane" books that first appeared in the 1930s. The books rely on word repetition, and pictures to support the meaning of the text. The idea is that if you see words enough, you eventually store them in your memory as visual images.</p>
<figure><img alt="Dick and Jane" src="https://img.apmcdn.org/c8383d637eace2f53f75d795ba70ca0a2449c218/uncropped/6e2fbc-20190820-dick-and-jane.jpg">
<figcaption><span>&nbsp;</span> <span>&nbsp;</span></figcaption>
</figure>
<p>The other idea is that reading requires knowledge of the relationships between sounds and letters. Children learn to read by sounding out words. This approach is known as phonics. It goes way back, popularized in the 1800s with the McGuffey readers.</p>
<div><img src="https://img.apmcdn.org/c8de3e651149802edda84446d893bdd8a7b5533c/uncropped/7c1b50-20190820-mcguffey-reader.png"></div>
<p>These two ideas &mdash; whole word and phonics &mdash; had been taking turns as the favored way to teach reading until Goodman came along with what came to be known among educators as the "three-cueing system."</p>
<p>In the cueing theory of how reading works, when a child comes to a word she doesn't know, the teacher encourages her to think of a word that makes sense and asks: Does it look right? Does it sound right? If a word checks out on the basis of those questions, the child is getting it. She's on the path to skilled reading.</p>
<p>Teachers may not know the term "three cueing," but they're probably familiar with "MSV." M stands for using meaning to figure out what a word is, S for using sentence structure and V for using visual information (i.e., the letters in the words). MSV is a cueing idea that can be traced back to the late Marie Clay, a developmental psychologist from New Zealand who first laid out her theories about reading in a dissertation in the 1960s.<a href="#footnote-content-6">6</a></p>
<p>Clay developed her cueing theory independently of Goodman, but they met several times and had similar ideas about the reading process. Their theories were based on observational research. They would listen to children read, note the kinds of errors they made, and use that information to identify a child's reading difficulties. For example, a child who says "horse" when the word was "house" is probably relying too much on visual, or graphic, cues. A teacher in this case would encourage the child to pay more attention to what word would make sense in the sentence.</p>
<p>Goodman and Clay believed that letters were the least reliable of the three cues, and that as people became better readers, they no longer needed to pay attention to all the letters in words. "In efficient word perception the reader relies mostly on the sentence and its meaning and some selected features of the forms of words," Clay wrote.<a href="#footnote-content-7">7</a>&nbsp;For Goodman, accurate word recognition was not necessarily the goal of reading. The goal was to comprehend text.<a href="#footnote-content-8">8</a>&nbsp;If the sentences were making sense, the reader must be getting the words right, or right enough.</p>
<p>These ideas soon became the foundation for how reading was taught in many schools. Goodman's three-cueing idea formed the theoretical basis of an approach known as "whole language" that by the late 1980s had taken hold throughout America.<a href="#footnote-content-9">9</a>&nbsp;Clay built her cueing ideas into a reading intervention program for struggling first-graders called Reading Recovery. It was implemented across New Zealand in the 1980s and went on to become one of the world's most widely used reading intervention programs.<a href="#footnote-content-10">10</a></p>
<p>But while cueing was taking hold in schools, scientists were busy studying the cognitive processes involved in reading words. And they came to different conclusions about how people read.<a href="#footnote-content-11">11</a></p>
<div>
<figure><img alt="Buddy reading" src="https://img.apmcdn.org/7be0cd043fa31d43ed8226bbd05560cb991fec58/uncropped/9bfba3-20190821-buddy-reading.jpg">
<figcaption><span>First-graders in Oakland, California, practice reading.</span> <span>Hasain Rasheed for APM Reports</span></figcaption>
</figure>
</div>
<h3>Scientists take on three cueing</h3>
<p>It was the early 1970s, and Keith Stanovich was working on his doctorate in psychology at the University of Michigan. He thought the reading field was ready for an infusion of knowledge from the "cognitive revolution" that was underway in psychology. Stanovich had a background in experimental science and an interest in learning and cognition due in part to the influence of his wife, Paula, who was a special education teacher.</p>
<p>Stanovich wanted to understand how people read words.<a href="#footnote-content-12">12</a>&nbsp;He knew about Goodman's work and thought he was probably right that as people become better readers, they relied more on their knowledge of vocabulary and language structure to read words and didn't need to pay as much attention to the letters.</p>
<p>So, in 1975, Stanovich and a fellow graduate student set out to test the idea in their lab. They recruited readers of various ages and abilities and gave them a series of word-reading tasks. Their hypothesis was that skilled readers rely more on contextual cues to recognize words than poor readers, who probably weren't as good at using context.</p>
<p>They couldn't have been more wrong.</p>
<p>"To our surprise, all of our research results pointed in the opposite direction," Stanovich wrote. "It was the poorer readers, not the more skilled readers, who were more reliant on context to facilitate word recognition."<a href="#footnote-content-13">13</a></p>
<p>The skilled readers could instantly recognize words without relying on context. Other researchers have confirmed these findings with similar experiments. It turns out that the ability to read words in isolation quickly and accurately is the hallmark of being a skilled reader. This is now one of the most consistent and well-replicated findings in all of reading research.<a href="#footnote-content-14">14</a></p>
<p>Other studies revealed further problems with the cueing theory:<a href="#footnote-content-15">15</a></p>
<ul>
<li>
<p>Skilled readers don't scan words and sample from the graphic cues in an incidental way; instead, they very quickly recognize a word as a sequence of letters. That's how good readers instantly know the difference between "house" and "horse," for example.</p>
</li>
<li>
<p>Experiments that force people to use context to predict words show that even skilled readers can correctly guess only a fraction of the words; this is one reason people who rely on context to identify words are poor readers.</p>
</li>
<li>
<p>Weak word recognition skills are the most common and debilitating source of reading problems.<a href="#footnote-content-16">16</a></p>
</li>
</ul>
<p>The results of these studies are not controversial or contested among scientists who study reading. The findings have been incorporated into every major scientific model of how reading works.</p>
<p>But cueing is still alive and well in schools.</p>
<h3>Picture Power!</h3>
<p>It's not hard to find examples of the cueing system. A quick search on Google, Pinterest or Teachers Pay Teachers turns up plenty of lesson plans, teaching guides and classroom posters. One popular poster has cute cartoon characters to remind children they have lots of strategies to use when they're stuck on a word, including looking at the picture (Eagle Eye), getting their lips ready to try the first sound (Lips the Fish), or just skipping the word altogether (Skippy Frog).</p>
<div>
<figure><img alt="Eagle Eye, Lips Fish, Skippy Frog" src="https://img.apmcdn.org/fb212db3c9cbaf3b5af6c3a1ae8cdefb6b79439b/uncropped/d37f84-20190821-eagle-eye-lips-fish-skippy-frog.png">
<figcaption><span>&nbsp;</span> <span>&nbsp;</span></figcaption>
</figure>
</div>
<p>There are videos online where you can see cueing in action. In one video posted on The Teaching Channel,<a href="#footnote-content-17">17</a>&nbsp;a kindergarten teacher in Oakland, California, instructs her students to use "picture power" to identify the words on the page. The goal of the lesson, according to the teacher, is for the students to "use the picture and a first sound to determine an unknown word in their book."</p>
<p>The class reads a book together called "In the Garden." On each page, there's a picture of something you might find in a garden. It's what's known as a predictable book; the sentences are all the same except for the last word.</p>
<figure><img alt="Look at the caterpillar." src="https://img.apmcdn.org/e12ca01c24d00bfcfbc5ced206555fb129105198/uncropped/7b4107-20190820-look-at-the-caterpillar.jpg">
<figcaption><span>&nbsp;</span> <span>&nbsp;</span></figcaption>
</figure>
<p>The children have been taught to memorize the words "look," "at," and "the." The challenge is getting the last word in the sentence. The lesson plan tells the teacher to cover up the word with a sticky note.</p>
<p>In the video, the wiggly kindergarteners sitting cross-legged on the floor come to a page with a picture of a butterfly. The teacher tells the kids that she's guessing the word is going to be butterfly. She uncovers the word to check on the accuracy of her guess.</p>
<p>"Look at that," she tells the children, pointing to the first letter of the word. "It starts with the /b/ /b/ /b/." The class reads the sentence together as the teacher points to the words. "Look at the butterfly!" they yell out excitedly.</p>
<p>This lesson comes from "Units of Study for Teaching Reading," more commonly known as "reader's workshop."<a href="#footnote-content-18">18</a>&nbsp;According to the lesson plan, this lesson teaches children to "know and apply grade-level phonics and word analysis skills in decoding words."<a href="#footnote-content-19">19</a></p>
<p>But the children were not taught to decode words in this lesson. They were taught to guess words using pictures and patterns &mdash; hallmarks of the three-cueing system.</p>
<p>The author of Units of Study for Teaching Reading, Lucy Calkins, often refers to cueing in her published work.<a href="#footnote-content-20">20</a>&nbsp;She uses the term MSV &mdash; the meaning, structure and visual idea that originally came from Clay in New Zealand.</p>
<p>Then there is Fountas and Pinnell Literacy, started by Irene Fountas and Gay Su Pinnell, teachers who learned the MSV concept from Clay in the 1980s.<a href="#footnote-content-21">21</a></p>
<p>Fountas and Pinnell have written several books about teaching reading, including a best-seller about a widely used instructional approach called "Guided Reading." They also created a reading assessment system that uses what are called "leveled books."<a href="#footnote-content-22">22</a>&nbsp;Children start with predictable books like "In the Garden" and move up levels as they're able to "read" the words. But many of the words in those books &mdash; butterfly, caterpillar &mdash; are words that beginning readers haven't been taught to decode yet. One purpose of the books is to teach children that when they get to a word they don't know, they can use context to figure it out.</p>
<p>When put into practice in the classroom, these approaches can cause problems for children when they are learning to read.</p>
<h3>'That is not reading'</h3>
<p>Margaret Goldberg, a teacher and literacy coach in the Oakland Unified School District, remembers a moment when she realized what a problem the three-cueing approach was. She was with a first-grader named Rodney when he came to a page with a picture of a girl licking an ice cream cone and a dog licking a bone.</p>
<p>The text said: "My little dog likes to eat with me."</p>
<p>But Rodney said: "My dog likes to lick his bone."</p>
<figure><img alt="My little dog" src="https://img.apmcdn.org/2f90f4f9e2a4462581cd6c7525669e52e4b6a719/uncropped/45f8e9-20190820-my-dog-likes-to-eat-with-me.png">
<figcaption><span>&nbsp;</span> <span>&nbsp;</span></figcaption>
</figure>
<p>Rodney breezed right through it, unaware that he hadn't read the sentence on the page.</p>
<p>Goldberg realized lots of her students couldn't actually read the words in their books; instead, they were memorizing sentence patterns and using the pictures to guess. One little boy exclaimed, "I can read this book with my eyes shut!"</p>
<p>"Oh no," Goldberg thought. "That is not reading."</p>
<p>Goldberg had been hired by the Oakland schools in 2015 to help struggling readers by teaching a Fountas and Pinnell program called "Leveled Literacy Intervention" that uses leveled books and the cueing approach.<a href="#footnote-content-23">23</a></p>
<p>Around the same time, Goldberg was trained in a program that uses a different strategy for teaching children how to read words. The program is called "Systematic Instruction in Phonological Awareness, Phonics, and Sight Words," or SIPPS.<a href="#footnote-content-24">24</a>&nbsp;It's a phonics program that teaches children how to sound out words and uses what are known as "decodable books." Most words in the books have spelling patterns that kids have been taught in their phonics lessons.</p>
<p>Goldberg decided to teach some of her students using the phonics program and some of her students using three cueing. And she began to notice differences between the two groups of kids. "Not just in their abilities to read," she said, "but in the way they approached their reading."</p>
<p>Goldberg and a colleague recorded first-graders talking about what makes them good readers.</p>
<p>One video shows Mia, on the left, who was in the phonics program. Mia says she's a good reader because she looks at the words and sounds them out. JaBrea, on the right, was taught the cueing system. JaBrea says: "I look at the pictures and I read it."</p>
<p data-url="https://www.youtube.com/watch?v=27E6kkNJY_o"></p>
<p><span>Courtesy of Margaret Goldberg, Oakland Unified School District</span></p>
<p>It was clear to Goldberg after just a few months of teaching both approaches that the students learning phonics were doing better. "One of the things that I still struggle with is a lot of guilt," she said.</p>
<p>She thinks the students who learned three cueing were actually harmed by the approach. "I did lasting damage to these kids. It was so hard to ever get them to stop looking at a picture to guess what a word would be. It was so hard to ever get them to slow down and sound a word out because they had had this experience of knowing that you predict what you read before you read it."</p>
<p>Goldberg soon discovered the decades of scientific evidence against cueing.<a href="#footnote-content-25">25</a>&nbsp;She was shocked. She had never come across any of this science in her teacher preparation or on the job.</p>
<p>And she started to wonder why not.</p>
<h3>Balanced Literacy</h3>
<p>People have been arguing for centuries about how children should be taught to read. The fight has mostly focused on whether to teach phonics.</p>
<p>The whole language movement of the late 20th century was perhaps the zenith of the anti-phonics argument.<a href="#footnote-content-26">26</a>&nbsp;Phonics instruction was seen as tedious, time-consuming and ultimately unnecessary. Why? Because &mdash; according to the three-cueing theory &mdash; readers can use other, more reliable cues to figure out what the words say.<a href="#footnote-content-27">27</a></p>
<p>Marilyn Adams came across this belief in the early 1990s. She's a cognitive and developmental psychologist who had just written a book summarizing the research on how children learn to read.<a href="#footnote-content-28">28</a>&nbsp;One big takeaway from the book is that becoming a skilled reader of English requires knowledge of sound-spelling correspondences.<a href="#footnote-content-29">29</a>&nbsp;Another big takeaway is that many kids were not being taught this in school.</p>
<p>Soon after the book was published, Adams was describing her findings to a group of teachers and state education officials in Sacramento, California. She was sensing discomfort and confusion in the room. "And I just stopped and said, 'What is it that I'm missing?'" she recalled. "'What is it that we need to talk about? Help me.'"</p>
<p>A woman raised her hand and asked: "What does this have to do with the three-cueing system?" Marilyn didn't know what the three-cueing system was. "I think I blew all of their fuses that I did not [know what it was] since this was so fundamental to being an elementary reading teacher," she said. "How could I present myself to them as an expert on reading and not know about this?"</p>
<p>The teachers drew her a Venn diagram of the three-cueing system. It looked something like this:</p>
<figure><img alt="venn" src="https://img.apmcdn.org/ccbe5d327ddcb469cd8d6e3b3f90994e55637a4c/uncropped/721b29-20190822-venn.png">
<figcaption><span>&nbsp;</span> <span>&nbsp;</span></figcaption>
</figure>
<p>Adams thought this diagram made perfect sense. The research clearly shows that readers use all of these cues to understand what they're reading.</p>
<p>But Adams soon figured out the disconnect. Teachers understood these cues not just as the way readers construct meaning from text, but as the way readers actually identify the words on the page. And they thought that teaching kids to decode or sound out words was not necessary.</p>
<p>"The most important thing was for the children to understand and enjoy the text," Adams said. "And from that understanding and joy of reading, the words on the page would just pop out at them."</p>
<p>She would explain to teachers at every opportunity that explicitly teaching children about the relationships between sounds and letters is essential to ensure all kids get off to a good start in reading. But she got tons of pushback from teachers. "They didn't want to teach phonics!" she recalled in frustration.</p>
<p>In 1998, Adams wrote a book chapter about how the three-cueing system conflicts with what researchers have figured out about reading. She hoped it would help put three cueing to rest.<a href="#footnote-content-30">30</a></p>
<p>By this time, the scientific research on reading was gaining traction. In 2000, a national panel convened by Congress to review the evidence on how to teach reading came out with a report.<a href="#footnote-content-31">31</a>&nbsp;It identified several essential components of reading instruction, including vocabulary, comprehension and phonics. The evidence that phonics instruction enhances children's success in learning how to read was clear and compelling. National reports on reading a few years later in the United Kingdom and Australia came to the same conclusion.<a href="#footnote-content-32">32</a></p>
<aside>
<blockquote>
<p><strong>"When I got into the classroom and someone told me to use this practice, I didn't question it."</strong></p>
<cite>-Stacey Cherny</cite>
</blockquote>
</aside>
<p>Eventually, many whole language supporters accepted the weight of the scientific evidence about the importance of phonics instruction. They started adding phonics to their books and materials and renamed their approach "balanced literacy."</p>
<p>But they didn't get rid of the three-cueing system.</p>
<p>Balanced literacy proponents will tell you their approach is a mix of phonics instruction with plenty of time for kids to read and enjoy books. But look carefully at the materials and you'll see that's not really what balanced literacy is mixing. Instead, it's mixing a bunch of different ideas about how kids learn to read. It's a little bit of whole word instruction with long lists of words for kids to memorize. It's a little bit of phonics. And fundamentally, it's the idea that children should be taught to read using the three-cueing system.</p>
<p>And it turns out cueing may actually prevent kids from focusing on words in the way they need to become skilled readers.</p>
<h3>Mapping the words</h3>
<aside>
<p>PREVIOUSLY</p>
<div>
<p>SEPT. 11, 2017</p>
<div><a href="https://www.apmreports.org/story/2017/09/11/hard-to-read"><img src="https://img.apmcdn.org/01e23d415b744e40bf21957d0762c73c3163fc24/widescreen/0db401-20170627-hard-to-read-how-american-schools-fail-kids-with-dyslexia.jpg"></a></div>
</div>
<div>
<p>SEPT. 10, 2018</p>
<div><a href="https://www.apmreports.org/story/2018/09/10/hard-words-why-american-kids-arent-being-taught-to-read"><img src="https://img.apmcdn.org/f46a87da9ccb2a7795e1b006a0f06adac87783f4/normal/082366-20180831-hard-words.jpg"></a></div>
</div>
</aside>
<p>To understand why cueing can get in the way of children's reading development, it's essential to understand how our brains process the words we see.</p>
<p>Reading scientists have known for decades that the hallmark of being a skilled reader is the ability to instantly and accurately recognize words.<a href="#footnote-content-33">33</a>&nbsp;If you're a skilled reader, your brain has gotten so good at reading words that you process the word "chair" faster than you process a picture of a chair.<a href="#footnote-content-34">34</a>&nbsp;You know tens of thousands of words instantly, on sight. How did you learn to do that?</p>
<p>It happens through a process called "orthographic mapping."<a href="#footnote-content-35">35</a>&nbsp;This occurs when you pay attention to the details of a written word and link the word's pronunciation and meaning with its sequence of letters.<a href="#footnote-content-36">36</a>&nbsp;A child knows the meaning and pronunciation of "pony." The word gets mapped to his memory when he links the sounds /p/&nbsp;/o/&nbsp;/n/&nbsp;/y/ to the written word "pony."</p>
<p>That requires an awareness of the speech sounds in words and an understanding of how those sounds are represented by letters.<a href="#footnote-content-37">37</a>&nbsp;In other words, you need phonics skills.</p>
<p>Here's what happens when a reader who has good phonics skills comes to a word she doesn't recognize in print. She stops at the word and sounds it out. If it's a word she knows the meaning of, she has now linked the spelling of the word with its pronunciation. If she doesn't know the meaning of the word, she can use context to try to figure it out.</p>
<p>By about second grade, a typically developing reader needs just a few exposures to a word through understanding both the pronunciation and the spelling for that word to be stored in her memory.<a href="#footnote-content-38">38</a>&nbsp;She doesn't know that word because she memorized it as a visual image. She knows that word because at some point she successfully sounded it out.</p>
<p>The more words she stores in her memory this way, the more she can focus on the meaning of what she's reading; she'll eventually be using less brain power to identify words and will be able to devote more brain power to comprehending what she's reading.<a href="#footnote-content-39">39</a></p>
<p>But when children don't have good phonics skills, the process is different.</p>
<p>"They sample from the letters because they're not good at sounding them out," said David Kilpatrick, a psychology professor at SUNY Cortland and the author of a book about preventing reading difficulties.<a href="#footnote-content-40">40</a>&nbsp;"And they use context."</p>
<p>In other words, when people don't have good phonics skills, they use the cueing system.</p>
<p>"The three-cueing system is the way poor readers read," said Kilpatrick.</p>
<p>And if teachers use the cueing system to teach reading, Kilpatrick says they're not just teaching children the habits of poor readers, they are actually impeding the orthographic mapping process.<a href="#footnote-content-41">41</a></p>
<p>"The minute you ask them just to pay attention to the first letter or look at the picture, look at the context, you're drawing their attention away from the very thing that they need to interact with in order for them to read the word [and] remember the word," Kilpatrick said. In this way, he said, three cueing can actually prevent the critical learning that's necessary for a child to become a skilled reader.</p>
<p>In many balanced literacy classrooms, children are taught phonics and the cueing system. Some kids who are taught both approaches realize pretty quickly that sounding out a word is the most efficient and reliable way to know what it is. Those kids tend to have an easier time understanding the ways that sounds and letters relate. They'll drop the cueing strategies and begin building that big bank of instantly known words that is so necessary for skilled reading.</p>
<p>But some children will skip the sounding out if they're taught they have other options. Phonics is challenging for many kids. The cueing strategies seem quicker and easier at first. And by using context and memorizing a bunch of words, many children can look like good readers &mdash; until they get to about third grade, when their books begin to have more words, longer words, and fewer pictures. Then they're stuck. They haven't developed their sounding-out skills. Their bank of known words is limited. Reading is slow and laborious and they don't like it, so they don't do it if they don't have to. While their peers who mastered decoding early are reading and teaching themselves new words every day, the kids who clung to the cueing approach are falling further and further behind.<a href="#footnote-content-42">42</a></p>
<p>These poor reading habits, once ingrained at a young age, can follow kids into high school. Some kids who were taught the cueing approach never become good readers. Not because they're incapable of learning to read well but because they were taught the strategies of struggling readers.</p>
<div>
<figure><img alt="Oakland teachers" src="https://img.apmcdn.org/94d4b25b7caf5ce3cbbda0fdb6e810cc9aa6ac0f/uncropped/422eae-20190819-oakland-teachers.jpg">
<figcaption><span>Margaret Goldberg (second from left) and Lani Mednick (right) are literacy coaches for the Oakland Unified School District. Dana Cilono (left) was with the charter school network Education for Change, and Erin Cox is with Aspire Public Schools. They are working on projects to rid their schools of the three-cueing system.</span> <span>Hasain Rasheed for APM Reports</span></figcaption>
</figure>
</div>
<h3>'So what if they use the picture?'</h3>
<p>Once Margaret Goldberg discovered the cognitive science evidence against cueing, she wanted her colleagues in the Oakland school district to know about it too.</p>
<p>Over the past two years, Goldberg and a fellow literacy coach named Lani Mednick have been leading a grant-funded pilot project to improve reading achievement in the Oakland schools.<a href="#footnote-content-43">43</a></p>
<p>They have their work cut out for them. Nearly half the district's third-graders are below grade level in reading. Goldberg and Mednick want to raise questions about how kids in Oakland are being <em>taught</em> to read.</p>
<p>They meet every couple weeks with literacy coaches from the 10 elementary schools in the pilot program. They read and discuss articles about the scientific research on reading. At a meeting in March, the coaches watched the video of the "picture power" lesson.</p>
<p>"This teacher meant well," Mednick said to the coaches after they watched the lesson. "It seemed like she believed this lesson would ensure her students would be on the road toward reading."</p>
<p>Mednick wanted the coaches to consider the beliefs about reading that would lead to the creation of a lesson like "picture power." The Oakland schools purchased the Units of Study for Teaching Reading series, which includes the "picture power" lesson, as part of a balanced literacy initiative the district began about 10 years ago. The district also bought the Fountas and Pinnell assessment system.</p>
<p>The coaches saw right away that "picture power" was designed to teach kids the cueing system. But they said many teachers don't see any problem with cueing. After all, one of the cues is to look at the letters in the word. What's wrong with teaching kids lots of different strategies to figure out unknown words?<a href="#footnote-content-44">44</a></p>
<p>"I remember before we started looking at the science and everything, I thought to myself, 'Reading is so hard for kids, so what if they use the picture?'" said Soraya Sajous-Brooks, the early literacy coach at Prescott Elementary School in West Oakland. "Like, use everything you've got."</p>
<p>But she's come to understand that cueing sends the message to kids that they don't need to sound out words. Her students would get phonics instruction in one part of the day. Then they'd go reader's workshop and be taught that when they come to a word they don't know, they have lots of strategies. They can sound it out. They can also check the first letter, look at the picture, think of a word that makes sense.</p>
<p>Teaching cueing <em>and</em> phonics doesn't work, Sajous-Brooks said. "One negates the other."</p>
<p>Goldberg and Mednick want to show the district there's a better way to teach reading. Schools in the pilot project used grant money to buy new materials that steer clear of the three-cueing idea. Two charter school networks in Oakland are working on similar projects to move their schools away from cueing.</p>
<p>To see what it looks like, I visited a first-grade classroom at a charter school in Oakland called Achieve Academy.<a href="#footnote-content-45">45</a></p>
<p>One part of the day was explicit phonics instruction.<a href="#footnote-content-46">46</a>&nbsp;The students were divided into small groups based on their skill level. They met with their teacher, Andrea Ruiz, at a kidney-shaped table in a corner of the classroom. The lowest-level group worked on identifying the speech sounds in words like "skin" and "skip." The highest-level group learned how verbs like "spy" and "cry" are spelled as "spied' and 'cried" in the past tense.</p>
<div>
<figure><img alt="Andrea Ruiz teaches phonics." src="https://img.apmcdn.org/464eccd7164b74cd08beab91d0f0ba043f31c8d4/uncropped/d55e93-20190819-andrea-ruiz-teaches-phonics.jpg">
<figcaption><span>Andrea Ruiz teaches a phonics lesson.</span> <span>Hasain Rasheed for APM Reports</span></figcaption>
</figure>
</div>
<p>There were also vocabulary lessons.<a href="#footnote-content-47">47</a>&nbsp;The entire class gathered on a rug at the front of the classroom to talk about a book Ms. Ruiz read out loud to them. One of the words in the book was "prey."</p>
<p>"What animals are a chameleon's prey?" Ms. Ruiz asked the children. "Or we can also ask, what animals do chameleons hunt for food?"</p>
<p>The kids turned and talked to each other. "A chameleon's prey are bugs and insects and other chameleons and mice and birds," a little boy explained to his classmate. "That's it."</p>
<p>Other vocabulary words these first-graders had learned were posted on cards around the classroom. They included: wander, persevere, squint and scrumptious. The kids weren't expected to read those words yet. The idea is to build their oral vocabulary so that when they <em>can</em> read those words, they know what the words mean.</p>
<p>This comes straight from the scientific research, which shows that reading comprehension is the product of two things.<a href="#footnote-content-48">48</a>&nbsp;First, a child needs to be able to sound out a word. Second, the child needs to know the meaning of the word she just sounded out. So, in a first-grade classroom that's following the research, you will see explicit phonics instruction and also lessons that build oral vocabulary and background knowledge. And you will see kids practicing what they've been taught.</p>
<p>After their vocabulary lesson, the kids did "buddy reading." They retreated to various spots around the classroom to read books to each other. I found Belinda sitting on an adult chair at the back of the classroom, her little legs swinging. Across from her was her buddy Steven, decked out in a yellow and blue plaid shirt tucked neatly into his jeans. He held the book and pointed to the words while Belinda read.</p>
<p>"Ellen /m/," Belinda paused, sounding out the word "meets." She was reading a decodable book about some kids who visit a farm. Almost all of the words in the book contain spelling patterns she'd been taught in her phonics lessons.</p>
<p>"I am a farm here," Belinda read.</p>
<p>Steven did a double-take. "A farmer here," he said gently. Steven's job as Belinda's reading buddy was to help her if she missed a word or got stuck. But that didn't happen much because Belinda had been taught how to read the words. She didn't need any help from the pictures, either. She barely glanced at them as she read.</p>
<div>
<figure><img alt="Steven and Belinda, buddy reading" src="https://img.apmcdn.org/9e4812e93a084b7a29c4b1b71202299fa210ae6a/uncropped/7aff27-20190821-steven-and-belinda-buddy-reading.jpg">
<figcaption><span>Steven and Belinda do "buddy reading."</span> <span>Hasain Rasheed for APM Reports</span></figcaption>
</figure>
</div>
<p>To be clear, there's nothing wrong with pictures. They're great to look at and talk about, and they can help a child comprehend the meaning of a story. Context &mdash; including a picture if there is one &mdash; helps us understand what we're reading all the time. But if a child is being taught to use context to identify words, she's being taught to read like a poor reader.</p>
<p>Many educators don't know this because the cognitive science research has not made its way into many schools and schools of education.<a href="#footnote-content-49">49</a></p>
<p>Ruiz didn't know about this research until the Oakland pilot project. "I didn't really know anything about how kids learn to read when I started teaching," she said. It was a relief when she came to Oakland and the curriculum spelled out that kids use meaning, structure and visual cues to figure out words. "Because I came from not having anything, I was like, 'Oh, there's a way we should teach this,'" she said.</p>
<p>I heard this from other educators. Cueing was appealing because they didn't know what else to do.</p>
<p>"When I got into the classroom and someone told me to use this practice, I didn't question it," said Stacey Cherny, a former teacher who's now principal of an elementary school in Pennsylvania. She says many teachers aren't taught what they need to know about the structure of the English language to be able to teach phonics well. She says phonics can be intimidating; three cueing isn't.</p>
<p>Another reason cueing holds on is that it seems to work for some children. But researchers estimate there's a percentage of kids &mdash; perhaps about 40 percent &mdash; who will learn to read no matter how they're taught.<a href="#footnote-content-50">50</a>&nbsp;According to Kilpatrick, children who learn to read with cueing are succeeding in spite of the instruction, not because of it.</p>
<p>Goldberg hopes the pilot project in Oakland will convince the district to drop all instructional materials that include cueing.</p>
<p>When asked about this, the Oakland superintendent's office responded with a written statement that there isn't enough evidence from the pilot project to make curriculum changes for the entire district and that the Oakland schools remain committed to balanced literacy.</p>
<p>Oakland's situation is no different from many other districts across the country that have invested millions of dollars in materials that include cueing.</p>
<p>"It feels like everyone's trusting somebody else to have done their due diligence," Goldberg said. "Classroom teachers are trusting that the materials they're being handed will work. The people who purchase the materials are trusting if they were on the market, that they will work. We're all trusting, and it's a system that is broken."</p>
<h3>'My science is different'</h3>
<p>If cueing was debunked decades ago by cognitive scientists, why is the idea still in materials that are being sold to schools?</p>
<p>One answer to that question is that school districts still buy the materials. Heinemann, the company that publishes the Fountas and Pinnell and Lucy Calkins' products that the Oakland schools use, earned somewhere in the neighborhood of $500 million in 2018, according to earnings reports.<a href="#footnote-content-51">51</a></p>
<p>I wanted to know what the authors of those materials make of the cognitive science research. And I wanted to give them a chance to explain the ideas behind their work. I wrote to Calkins, Fountas and Pinnell and asked for interviews. They all declined. Heinemann sent a statement that said every product the company sells is informed by extensive research.</p>
<p>I also asked Ken Goodman for an interview. It's been more than 50 years since he first laid out the three-cueing theory in that 1967 paper. I wanted to know what he thinks of the cognitive science research. Of the major proponents of three cueing I reached out to, he was the only one who agreed to an interview.</p>
<p>I visited Goodman at his home in Tucson, Arizona. He's 91. He uses a scooter to get around, but he's still working. He just finished a new edition of one of his books.</p>
<div>
<figure><img alt="Ken Goodman" src="https://img.apmcdn.org/2c403db3213fd2c5179719cb1c6b969eccc35657/uncropped/a75456-20190819-goodmans.jpg">
<figcaption><span>Ken Goodman with his wife and frequent co-author, Yetta Goodman.</span> <span>Emily Hanford | APM Reports</span></figcaption>
</figure>
</div>
<p>When I asked him what he makes of the cognitive science research, he told me he thinks scientists focus too much on word recognition. He still doesn't believe accurate word recognition is necessary for reading comprehension.</p>
<p>"Word recognition is a preoccupation," he said. "I don't teach word recognition. I teach people to make sense of language. And learning the words is incidental to that."</p>
<p>He brought up the example of a child who comes to the word "horse" and says "pony" instead. His argument is that a child will still understand the meaning of the story because horse and pony are the same concept.</p>
<p>I pressed him on this. First of all, a pony isn't the same thing as a horse. Second, don't you want to make sure that when a child is learning to read, he understands that /p/&nbsp;/o/&nbsp;/n/&nbsp;/y/ says "pony"? And different letters say "horse"?</p>
<p>He dismissed my question.</p>
<p>"The purpose is not to learn words," he said. "The purpose is to make sense."</p>
<p>Cognitive scientists don't dispute that the purpose of reading is to make sense of the text. But the question is: How can you understand what you are reading if you can't accurately read the words? And if quick and accurate word recognition is the hallmark of being a skilled reader, how does a little kid get there?</p>
<p>Goodman rejected the idea that you can make a distinction between skilled readers and unskilled readers; he doesn't like the value judgment that implies. He said dyslexia does not exist &mdash; despite lots of evidence that it does.<a href="#footnote-content-52">52</a>&nbsp;And he said the three-cueing theory is based on years of observational research. In his view, three cueing is perfectly valid, drawn from a different kind of evidence than what scientists collect in their labs.</p>
<p>"My science is different," Goodman said.</p>
<p>This idea that there are different kinds of evidence that lead to different conclusions about how reading works is one reason people continue to disagree about how children should be taught to read. It's important for educators to understand that three cueing is based on theory and observational research and that there's decades of scientific evidence from labs all over the world that converges on a very different idea about skilled reading.</p>
<p>The cognitive science does not provide all the answers about how to teach children to read, but on the question of how skilled readers read words, scientists have amassed a huge body of evidence.</p>
<p>Goldberg thinks it's time for educators across the country to take a close look at all the materials they use to teach reading.</p>
<p>"We should look through the materials and search for evidence of cueing," she said. "And if it's there, don't touch it. Don't let it get near our kids, don't let it get near our classrooms, our teachers."</p>
<div>
<div>
<div>
<p>At a Loss for Words is one of three audio documentaries this season from the</p>
<a href="https://www.apmreports.org/educate-podcast">Educate podcast</a>
<p>&mdash; stories about education, opportunity, and how people learn.</p>
</div>
</div>
</div>
<p><strong>Share and discuss on Facebook</strong></p>
<hr>
<div>
<div>
<p><strong>EXECUTIVE EDITOR</strong><br>
Stephen Smith</p>
<p><strong>EDITOR</strong><br>
Catherine Winter</p>
</div>
<div>
<p><strong>ASSOCIATE PRODUCER</strong><br>
Alex Baumhardt</p>
<p><strong>PRODUCTION HELP</strong><br>
John Hernandez</p>
</div>
<div>
<p><strong>WEB EDITORS</strong><br>
Andy Kruse<br>
Dave Mann</p>
<p><strong>AUDIO MIX</strong><br>
Craig Thorson<br>
Chris Julin</p>
</div>
<div>
<p><strong>EDITOR-IN-CHIEF</strong><br>
Chris Worthington</p>
<p><strong>PROJECT COORDINATOR</strong><br>
Shelly Langford</p>
</div>
<div>
<p><strong>THEME MUSIC</strong><br>
Gary Meister</p>
<p><strong>FACT CHECKER</strong><br>
Betsy Towner Levine</p>
</div>
<div>
<p><strong>COPY EDITOR</strong><br>
Sherri Hildebrandt</p>
<p><strong>SPECIAL THANKS</strong><br>
Sasha Aslanian<br>
Heena Srivastava</p>
</div>
</div>
<p>Support for this program comes from the Spencer Foundation and Lumina Foundation.</p>
<p><img src="https://www.apmreports.org/assets/img_logo-spencer-9e0f2cd984fed846e62a2627e1ad1e10d302390f02385b090b69c9d252003425.jpg"><img src="https://www.apmreports.org/assets/img_logo-lumina-62f20d10ea882b985860f4cea5355ed999c2b8d3119ced490d807e02136dd558.png"></p>
<hr>
<p><strong>Feedback</strong></p>
<div>
<p>We're interested in hearing what impact APM Reports programs have on you. Has one of our documentaries or podcasts changed how you think about an issue? Has it led you to do something, like start a conversation or try to do something new in your community?</p>
<a href="https://mpr.tfaforms.net/6?tfa_187=APM&amp;tfa_236=Climate+change%2Fenvironment%2F+sustainability%2F+weather&amp;tfa_235=News+Services&amp;tfa_234=National+Arts+and+Ideas&amp;tfa_106=External+Webform">Share your impact story</a>
<p>.</p>
</div>
<p><strong>Resources</strong></p>
<hr>
<p><strong>Sign up for email notifications</strong></p>
<p>Enter your address below and we'll let you know when we publish new stories.</p>
</div>
</div>
<br>
<br>
<br>
<br>
<br>
<br>
via www.apmreports.org https://ift.tt/2wyeStF<br>
<br>
July 29, 2020 at 02:59PM
